owl.opam

```
@@ -27,7 +27,7 @@ depends: [
   "base" {build}
   "base-bigarray"
   "conf-openblas" {>= "0.2.0"}
-  "ctypes" {< "0.17.0"}
+  "ctypes"
```

Doesn’t this require the latest ctypes?
```suggestion
  "ctypes" { >= "0.17.0"}
```

---

these changes seem to work with older versions of ctypes as well. I tested it on 0.16.0

---

Let’s use 0.16.0 as a bound then. We can alwys relax it if really needed, but it will reduce testing burden to us and for the opam releases